server/item/totem.go

```diff
@@ -0,0 +1,13 @@
+package item
+
+type Totem struct{}
```

Comment:
I'd just document this struct, explain what the item is. Doesn't need to be much